### PR TITLE
goaccess: update to 1.4

### PR DESCRIPTION
**Pull request by alm0ndiga**
Hi all.
I am using OpenIndiana for some time and i would like contribute. This should be my first pull request. I am ready to do any amendments.
Thanks.

### CODE REVIEW

File: components/web/goaccess/goaccess.p5m
Comment (Toasterson) [inline components/web/goaccess/goaccess.p5m:25]: This got deleted during the update of the manifest. Please leave existing manual additions in when updating.

File: components/web/goaccess/goaccess.p5m
Comment (Toasterson) [inline components/web/goaccess/goaccess.p5m]: Directory actions are only required, if no other files below are owned by this package. The directories will be created automatically if needed. Please remove.

File: components/web/goaccess/goaccess.p5m
Comment (Toasterson) [reply components/web/goaccess/goaccess.p5m:25]: Please also make sure that config file locations don't change unless absolutely necessary. Existing users will need to be notified about the update otherwise.

File: components/web/goaccess/manifests/sample-manifest.p5m
Comment (Toasterson) [inline components/web/goaccess/manifests/sample-manifest.p5m:26]: Please include preserve=true to config files the user should be able to update, otherwise a "pkg fix" call to restore system state will overwrite user changes.

**Review by Toasterson — CHANGES_REQUESTED** [review]
See my comments otherwise LGTM

File: components/web/goaccess/goaccess.p5m
Comment (AndWac) [inline components/web/goaccess/goaccess.p5m:15]: 2020 :)

File: components/web/goaccess/goaccess.p5m
Comment (alm0ndiga) [reply components/web/goaccess/goaccess.p5m:25]: Thank you Toasterson, it's fixed, now the previous config file is there with preserve=true.

File: components/web/goaccess/goaccess.p5m
Comment (alm0ndiga) [reply components/web/goaccess/goaccess.p5m]: Thanks, it's removed now.

File: components/web/goaccess/manifests/sample-manifest.p5m
Comment (alm0ndiga) [reply components/web/goaccess/manifests/sample-manifest.p5m:26]: Thanks Toasterson, now it's included preserve=true in config files

File: components/web/goaccess/goaccess.p5m
Comment (alm0ndiga) [reply components/web/goaccess/goaccess.p5m:15]: Thanks AndWac, it's fixed now, i absolutely forget the year !!! :D

File: components/web/goaccess/manifests/sample-manifest.p5m
Comment (AndWac) [inline components/web/goaccess/manifests/sample-manifest.p5m:27]: Never touch the sample-manifest. Only generate it and add to your PR if it has changes other than the year.
With this you (or any other maintainer) can check whether something has changed since the last release.

File: components/web/goaccess/Makefile
Comment (AndWac) [inline components/web/goaccess/Makefile]: You need to remove this line. At least on my build system it is responsible for the $(MACH32) in the file action:
file path=usr/bin/$(MACH32)/goaccess

**Review by Toasterson — APPROVED** [review]

**Comment by alm0ndiga** [comment]
Great! Thanks a lot AndWac and Toasterson for make it possible :)

**Comment by Toasterson** [comment]
You only got one of two approvals. Look in the thread above for @AndWac comments. Those you two things you will still need to fix.

**Comment by alm0ndiga** [comment]
Thanks Toasterson, I've deleted right now that line in Makefile in order to avoid MACH32 line in sample-manifest.p5m.

**Comment by alm0ndiga** [comment]
Here, I've updated sample-manifest.p5m to original as generated by 'gmake sample-manifest'

File: components/web/goaccess/goaccess.p5m
Comment (AndWac) [inline components/web/goaccess/goaccess.p5m]: Did you run "gmake publish"? On my machine it fails:
Couldn't find 'etc/goaccess.conf' in any of the specified search directories:
	/oibuild/oi-userland/components/web/goaccess
	/oibuild/oi-userland/components/web/goaccess/build
	/oibuild/oi-userland/components/web/goaccess/build/prototype/i386
	/oibuild/oi-userland/components/web/goaccess/build/prototype/i386/mangled
	/oibuild/oi-userland/components/web/goaccess/goaccess-1.4
gmake: *** [/oibuild/oi-userland/make-rules/ips.mk:434: /oibuild/oi-userland/components/web/goaccess/build/manifest-i386-goaccess.depend] Error 1

Comment (AndWac) [reply components/web/goaccess/goaccess.p5m]: So, your idea with saving the old goaccess.conf file is fine but doesn't work as it is now. I propose to remove the line and warn possible users over the mailing list resp. release notes.
It is probably possible to do what you intended to do by means of bypass-generate  but I wouldn't do that here.

**Review by Toasterson — CHANGES_REQUESTED** [review]
Hi, @alm0ndiga

Sorry, I just looked into the goaccess configs, and they have changed sufficiently that we need to remove the preserve=true. Also `goaccess.conf` has moved from `etc` to `etc/goaccess` so we should only deliver the new config file this update.

**Comment by alm0ndiga** [comment]
I've modified goaccess.p5m deleting old config file

**Review by AndWac — APPROVED** [review]

**Comment by AndWac** [comment]
Thank you for your valuable work and congratulations to your first contribution!

**Pull request merged**
